rllib/agents/dqn/dqn_policy.py

```diff
@@ -177,7 +149,7 @@ def postprocess_trajectory(policy,
         policy.parameter_noise_sigma.load(
             policy.parameter_noise_sigma_val, session=policy.get_session())
 
-    return _postprocess_dqn(policy, sample_batch)
+    return postprocess_fn(policy, sample_batch)
```

Comment:
Consider `postprocess_n_step` for clarity.

Reply:
It does other things, though, as well.

Reply:
Good point. How about `postprocess_nstep_and_prio` then?

Reply:
:+1: